### DISCUSSION TITLE
Differential backup

### DISCUSSION THREAD

**Discussion by Akhi0587** [olahallengren/sql-server-maintenance-solution#558] [Questions & Answers]
Dears,  Do we have any scripts to calculate the size of differential backup? Let’s say full backup completed at 10:00am and want to take differential at 6:00pm, need to know the size of the differential before triggering the backup.

**Comment by olahallengren** [comment]
You can check it using the column modified_extent_page_count in sys.dm_db_file_space_usage.
https://docs.microsoft.com/en-us/sql/relational-databases/system-dynamic-management-views/sys-dm-db-file-space-usage-transact-sql
